devops/github-pages.md

```diff
@@ -0,0 +1,16 @@
+# GitHub Pages Deployment
+How to host Good sites on GitHub pages.
+
+[![good](http://builtbygood.co/img/logo.png)](http://builtbygood.co)
```

Comment:
So just have it on the main readme or maybe at the most the main readme
files for each section?

-- 
J.C. Hiatt
CEO, *Good* Design & Code
http://builtbygood.co
@builtbygood

On May 20, 2016 at 10:38:10 AM, Christian Hall (notifications@github.com)
wrote:

> In devops/github-pages.md
> <https://github.com/builtbygood/good-guides/pull/3#discussion_r64061716>:
>
> > @@ -0,0 +1,16 @@
> > +# GitHub Pages Deployment
> > +How to host Good sites on GitHub pages.
> > +
> > +[![good](http://builtbygood.co/img/logo.png)](http://builtbygood.co)
>
> There are a lot of pages, so I'd probably say it's overkill
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/builtbygood/good-guides/pull/3/files/f1fc6c0713bf0cae050a917881fa86e82dfee12e#r64061716>
>